scripts/deploy.ts

```diff
@@ -35,6 +36,7 @@ export const deploy = async (
     hre.ethers.BigNumber.from(args.maxsupply),
     hre.ethers.BigNumber.from(args.globalwalletlimit),
     args.cosigner ?? hre.ethers.constants.AddressZero,
+    args.timestampexpiryseconds,
```

Comment:
maybe we can provide a default of 300 seconds here

Reply:
Updated